patch_parser.py

```diff
@@ -348,12 +350,17 @@ def parse_comments(email_thread: Message) -> Patchset:
         comments = []
         for reply in patch.children:
             comments.extend(diff_reply(patch, reply))
-        set_index, length = parse_set_index(patch)
-        assert length == len(patches)
-        text = 'From: {from_}\nSubject: {subject}\n\n{content}'.format(
-                from_=patch.from_, subject=patch.subject, content=patch.content)
-        patch_list.append(Patch(text=patch.content, text_with_headers=text, set_index=set_index, comments=comments))
-        patch_list.sort(key=lambda x: x.set_index)
+        if (email_thread.in_reply_to):
```

Comment:
What does this new block of code do?

Reply:
I realize now that it doesn't handle cover letters anymore. But, I was trying to handle single email threads (with only the patch being sent).

Reply:
Might be a good idea to add a small unit test for this function.